docs/_docs/configuration/webrick.md

```diff
@@ -18,3 +18,16 @@ Jekyll provides by default `Content-Type` and `Cache-Control` response
 headers: one dynamic in order to specify the nature of the data being served,
 the other static in order to disable caching so that you don't have to fight
 with Chrome's aggressive caching when you are in development mode.
+
+
+You can override the default `Cache-Control` by setting it to a value of your
+own choosing (a caching instruction, or an empty string). If you don't want the WEBrick server
```

Comment:
Inconsistent line-lengths on this paragraph.
If the previous line ended at `your`, why is this line ending after three more words?